examples/intersection_observer/README.md

```diff
@@ -0,0 +1,11 @@
+# Intersection Observer Example
+
+This example uses the [Intersection Observer V1 API](https://developer.mozilla.org/en-US/docs/Web/API/Intersection_Observer_API) through web-sys.
```

Comment:
```suggestion
This example uses the [Intersection Observer API](https://developer.mozilla.org/en-US/docs/Web/API/Intersection_Observer_API) through `web-sys`.
```
Why `V1`?

Reply:
I removed it. I wanted to distinguish between the unofficial V2 https://caniuse.com/intersectionobserver-v2